plat/mediatek/mt6795/aarch64/platform_common.c

```diff
@@ -0,0 +1,148 @@
+/*
```

Comment:
We will do more code sharing with other platform, put common code in mediatek/common/platform_common.c